extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/hll/HllSketchBuildAggregator.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.aggregation.datasketches.hll;
+
+import com.yahoo.sketches.hll.HllSketch;
+import com.yahoo.sketches.hll.TgtHllType;
+
+import io.druid.java.util.common.IAE;
+import io.druid.query.aggregation.Aggregator;
+import io.druid.segment.ColumnValueSelector;
+
+/**
+ * This aggregator builds sketches from raw data.
+ * The input column can contain identifiers of type string, char[], byte[] or any numeric type.
+ * @author Alexander Saydakov
+ */
+public class HllSketchBuildAggregator implements Aggregator
+{
+
+  private final ColumnValueSelector<Object> selector;
+  private HllSketch sketch;
+
+  public HllSketchBuildAggregator(final ColumnValueSelector<Object> selector, final int lgK, final TgtHllType tgtHllType)
+  {
+    this.selector = selector;
+    sketch = new HllSketch(lgK, tgtHllType);
```

Comment:
nit: `this.sketch = ...` to be consistent with other assignment in constructors

Reply:
OK